src/AppLayout/AppLayout.js

```diff
@@ -154,23 +154,14 @@ AppLayout.propTypes = {
    * App version to display in header.
```

Comment:
I know this is not part of this PR. I can you have updated the wrong documentation in AppHeader.js . It would be nice if you can update the same here as well.